MIDAS/src/rocket_state.h

```diff
@@ -37,6 +37,7 @@ struct RocketState {
     bool pyro_should_be_firing;
 
     SensorState<LowGData> low_g;
+    SensorState<LowGData> backup_low_g;
```

Comment:
The primary purpose of this sensor is to now act at a pure gryo, and maybe that wasn't clear enough on my part. What I think you should do instead of creating another instance of the lowgdata struct is make a new struct to store specifically gyro data. Also, I would rename the sensor to something like gryo sensor. But other than the naming garbage this looks really good!

Reply:
the gyro struct would just have fields of gx gy and gz like the other accelerometer structs right now. We are planning to change the fields in the accelerometer structs to ax ay and az.

Reply:
so is lowgdata low g accelerometer?

Reply:
wah wah wah

Reply:
fixing it

Reply:
yes lowgdata is specifically acceleration values. This sensor is now just gonna give us gryo unless some shit goes really bad with the new lowg sensor and we have to scrap it completely.

Reply:
for the structs for lowg and highg, doing ax, ay, az would make more sense. i think gx, gy, gz, gives intuitive assumption that you're talking abt gyro but thats smol (not related to this pr <3)

Reply:
yeah it was supposed to be ax ay az it was just a small oopsie

Reply:
BIG OOPSIE

Reply:
LARGEST OOPSIE